src/main/java/net/test/interceptor/TimingInterceptorMs.java

```diff
@@ -33,8 +38,11 @@ public Object intercept(@Origin Method method, @SuperCall Callable<?> callable)
     try {
       return callable.call();
     } finally {
-      System.out.println(
-        commonActionArgs.addPrefix("TraceAgent (timing): `" + method + "` took " + (System.currentTimeMillis() - start) + " ms"));
+      long end = System.currentTimeMillis();
```

Comment:
```java
long end = this.logThresholdMs == 0) ? 0 : System.currentTimeMillis()
```

Reply:
Same as with the TimingInterceptorNano I don't think we need this change on the Timing interceptors as we are using the `end` variable anyways as part of the printout.

Reply:
... and this too